### PR TITLE
feat(apps/interface): remove cypress

### PR DESCRIPTION
**Pull request by asaadam**
remove cypress

Fix RIS-524

### CODE REVIEW

**Comment by pyk** [comment]
@asaadam please remove the `apps/interface/cypress` directory too

**Review by pyk — CHANGES_REQUESTED** [review]
We need to remove `cypress` directory too

File: apps/interface/package.json [hunk 1]
```diff
@@ -60,13 +58,5 @@
         "mock-match-media": "0.3.0",
         "start-server-and-test": "1.14.0",
```
Comment (pyk) [inline apps/interface/package.json:59]: @asaadam I think we need to remove `"start-server-and-test"` package  too coz we only use it for cypress

**Review by pyk — CHANGES_REQUESTED** [review]

File: apps/interface/package.json (hunk 1 above)
Comment (asaadam) [reply apps/interface/package.json:59]: okay ser

Comment (asaadam) [reply apps/interface/package.json]: updated ser, i also remove cypress script

**Review by pyk — APPROVED** [review]
Nice. Thanks @asaadam 🔥

**Pull request merged**